src/GovUk.Education.ExploreEducationStatistics.Content.Api/Services/ContentCacheService.cs

```diff
@@ -0,0 +1,95 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Content.Api.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.ViewModels;
+using Microsoft.Azure.Storage.Blob;
+using Newtonsoft.Json;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Api.Services
+{
+    public class ContentCacheService : IContentCacheService
+    {
+        private readonly  CloudBlobClient _cloudBlobClient;
+        private const string ContainerName = "cache";
+
+        public ContentCacheService(CloudBlobClient cloudBlobClient)
+        {
+            _cloudBlobClient = cloudBlobClient;
+        }
+        
+        public async Task<List<ThemeTree>> GetContentTreeAsync()
+        {
+            var container = await GetCloudBlobContainer();
+
+            var blob = container.GetBlockBlobReference($"publications/tree.json");
+
+            return !blob.Exists() ? null : JsonConvert.DeserializeObject<List<ThemeTree>>(blob.DownloadText());
+        }
+        
+        public async Task<List<ThemeTree>> GetMethodologyTreeAsync()
+        {
+            var container = await GetCloudBlobContainer();
+
+            var blob = container.GetBlockBlobReference($"methodology/tree.json");
```

Comment:
as previous

Reply:
Removed two other occurrences in `ContentCacheGenerationService` :+1: